CHANGELOG.md

```diff
@@ -43,9 +43,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - [#36](https://github.com/Kashoo/synctos/issues/36): Does not return a non-zero exit status when sync function generation fails
 
-[Unreleased]: https://github.com/Kashoo/synctos/compare/v1.4.0...HEAD
+## [1.0.0] - 2016-07-12
+First public release
+
+[Unreleased]: https://github.com/Kashoo/synctos/compare/v1.5.0...HEAD
+[1.5.0]: https://github.com/Kashoo/synctos/compare/v1.4.0...v1.5.0
```

Comment:
The URL for this comparison will not be valid until the `v1.5.0` tag is created, which will immediately follow merging this PR.